chart/flux/values.yaml

```diff
@@ -21,6 +21,8 @@ helmOperator:
   repository: quay.io/weaveworks/helm-operator
   tag: 0.5.3
   pullPolicy: IfNotPresent
+  # Limit the operator scope to a single namespace
+  namespace:
```

Comment:
should this be `namespace: ""`?

Reply:
If the namespace is not set then the default from the `pflag` is applied. I've added the namespace in the values.yaml to document the option not to provide a default value for it.